lib/connectors/memory.js

```diff
@@ -461,6 +460,15 @@ function applyFilter(filter) {
     if (typeof value === 'string' && (example instanceof RegExp)) {
       return value.match(example);
     }
+
+    if (example.regexp) {
+      var normalizedRegExp = utils.toRegExp(example.regexp);
+      if (normalizedRegExp instanceof Error)
```

Comment:
This should not happen in the connector. If the regexp operand is invalid, the juggler should have reported the error.